src/Impostor.Api/Innersloth/GameVersion.cs

```diff
@@ -1,20 +1,84 @@
+using System;
+using System.Numerics;
+
 namespace Impostor.Api.Innersloth
 {
-    public static class GameVersion
+    public readonly struct GameVersion : IEquatable<GameVersion>, IComparable<GameVersion>, IComparisonOperators<GameVersion, GameVersion, bool>
     {
-        public static int GetVersion(int year, int month, int day, int revision = 0)
+        public GameVersion(int value)
+        {
+            Value = value;
+        }
+
+        public GameVersion(int year, int month, int day, int revision = 0)
+        {
+            Value = (year * 25000) + (month * 1800) + (day * 50) + revision;
+        }
+
+        public int Value { get; }
+
+        public static bool operator ==(GameVersion left, GameVersion right) => left.Value == right.Value;
+
+        public static bool operator !=(GameVersion left, GameVersion right) => left.Value != right.Value;
+
+        public static bool operator >(GameVersion left, GameVersion right) => left.Value > right.Value;
+
+        public static bool operator >=(GameVersion left, GameVersion right) => left.Value >= right.Value;
+
+        public static bool operator <(GameVersion left, GameVersion right) => left.Value < right.Value;
+
+        public static bool operator <=(GameVersion left, GameVersion right) => left.Value <= right.Value;
+
+        public void GetComponents(out int year, out int month, out int day, out int revision)
+        {
+            var value = Value;
+            year = value / 25000;
+            value %= 25000;
+            month = value / 1800;
+            value %= 1800;
+            day = value / 50;
+            revision = value % 50;
+        }
+
+        public GameVersion ExtractDisableServerAuthority(out bool disableServerAuthority)
```

Comment:
this method is doing two things, it may be cleaner if you split this up into two items:
- normalizing the version to remove the dsa flag
- getting whether a specific version has the dsa flag set